phinx.json

```diff
@@ -0,0 +1,36 @@
+{
+  "paths": {
+    "migrations": "%%PHINX_CONFIG_DIR%%/migrations"
```

Comment:
I don't mean interactively generate it, I mean transmute the default PHP config into each type on demand. For instance, if I request YAML config:

- load the PHP config
- convert to YAML
- write config file

Repeat the same process for JSON. This prevents the YAML/JSON from getting out of sync with PHP.